v1/0_introduction.md

```diff
@@ -1,27 +1,44 @@
 
-The protocol operates through a series of carefully orchestrated steps. It begins with funding transactions that lock coins in 2-of-2 multisig outputs, followed by contract transactions that establish time-locked redemption conditions. Hash preimages serve as cryptographic proofs ensuring atomic execution, and the process concludes with a private key handover that completes the swap.
+## Scope of the project.
+While initially designed to ["massively improve Bitcoin fungibility"](https://gist.github.com/chris-belcher/9144bd57a91c194e332fb5ca371d0964), Coinswap has broader implications for the
+Bitcoin ecosystem. Decentralized cross-chain atomic swaps could enhance scalability by facilitating interoperability between Bitcoin and other UTXO-based sidechains like
+Liquid or Spacechain. In high-fee environments, Coinswap could enable users to swap into low fee sidechains for payments and later return to Bitcoin L1 cold storage. For federated sidechains like Liquid, this eliminates the reliance on PegIn/PegOut mechanisms, offering a seamless SwapIn/SwapOut experience and enhancing market liquidity.
 
-## Protocol Variants and Implementation
+Atomic swaps are not new. Services like [Boltz](https://boltz.exchange/) already provide swaps between Bitcoin and Lightning using [submarine swaps](https://docs.lightning.engineering/the-lightning-network/multihop-payments/understanding-submarine-swaps) and are exploring Bitcoin-to-Liquid atomic swaps with similar HTLC contracts. However, all existing cross-chain swap solutions are centralized, compromising user privacy and exposing funds to theft or seizure. Coinswap’s fully decentralized design addresses these vulnerabilities.
 
-Coinswap can be implemented in two main forms, each serving different privacy needs. The simpler two-party swaps enable direct exchanges between participants, while multi-party routed swaps enhance privacy through multiple makers in a system similar to onion routing. This flexibility allows users to choose the implementation that best suits their privacy requirements and transaction needs.
+The Coinswap protocol employs a "smart-client, dumb-server" model. `Taker` clients coordinate the swap process, communicating with multiple `Maker` servers. Makers only respond to protocol messages from Takers and have no visibility into the full swap route or communication with other Makers. This lightweight, scalable architecture minimizes operational complexity for Makers, so node runners can run it minimal capital. To prevent fraud, the protocol incorporates [fidelity bonds](/v1/4_fidelity.md), and both Maker and Taker applications include built-in watchtower functionality for recovery in case of malicious behavior. The protocol exclusively communicates over Tor to preserve privacy, and a simple `DNS` server facilitates market discovery. Future releases will explore more decentralized DNS solutions.
```

Comment:
```suggestion
The Coinswap protocol employs a "smart-client, dumb-server" model. `Taker` clients coordinate the swap process, communicating with multiple `Maker` servers. Makers only respond to protocol messages from Takers and have no visibility into the full swap route or communication with other Makers. This lightweight, scalable architecture minimizes operational complexity for Makers, so node runners can run it with minimal capital. To prevent fraud, the protocol incorporates [fidelity bonds](/v1/4_fidelity.md), and both Maker and Taker applications include built-in watchtower functionality for recovery in case of malicious behavior. The protocol exclusively communicates over Tor to preserve privacy, and a simple `DNS` server facilitates market discovery. Future releases will explore more decentralized DNS solutions.
```